lifecycle-operator/controllers/lifecycle/keptntask/controller.go

```diff
@@ -36,10 +36,11 @@ import (
 // KeptnTaskReconciler reconciles a KeptnTask object
 type KeptnTaskReconciler struct {
 	client.Client
-	Scheme      *runtime.Scheme
-	EventSender controllercommon.IEvent
-	Log         logr.Logger
-	Meters      apicommon.KeptnMeters
+	Scheme            *runtime.Scheme
+	EventSender       controllercommon.IEvent
+	Log               logr.Logger
+	DefaultTTLSeconds int32
```

Comment:
We can leverage the K8s API to provide a default value so we don't need to have any extra code logic to handle that :) Please, clean up the other files from this default.
Nevertheless, great work on how you handle that 👍